shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -662,7 +664,10 @@ public void detachFromFlutterEngine() {
             + " connection to the engine "
             + getFlutterEngine()
             + " evicted by another attaching activity");
-    release();
```

Comment:
what about adding a test to ensure that `detachFromFlutterEngine()` doesn't release the engine?

Reply:
done.